prac_07/guitar.py

```diff
@@ -0,0 +1,21 @@
+class Guitar:
+    """Represent a guitar with its name, year of manufacture, and cost."""
+
+    def __init__(self, name="", year=0, cost=0):
+        """Initialize a Guitar instance with name, year, and cost."""
+        self.name = name
+        self.year = year
+        self.cost = cost
+
+    def get_age(self):
+        """Calculate the age of the guitar."""
+        current_year = 2022  # Replace with dynamic year if needed

```

Comment:
Is current year a variable or a CONSTANT? Should a constant be defined inside a method or aren't all constants supposed to be global?